.reaction/graphql-linter/util/listIsAlphabetical.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.listIsAlphabetical = listIsAlphabetical;
+
+/**
+ * @summary Returns `true` if two arrays have the same item values in the same order.
+ */
+function arraysEqual(a, b) {
+  for (var i = 0; i < a.length; ++i) {
+    if (a[i] !== b[i]) return false;
+  }
+  return true;
+}
+
+/**
+ * @summary Returns `true` if the list is in alphabetical order,
+ *   or an alphabetized list if not
+ * @param {String[]} list Array of strings
+ */
+function listIsAlphabetical(list) {
+  const sortedList = list.slice().sort();
+  if (arraysEqual(list, sortedList)) return true;
```

Comment:
The return types seem inconsistent `true` and array. Do we want just `return arraysEqual(list, sortedList);`?

Reply:
Oh hmm. I guess I don't really like this API. I expect an function with `is` in the name to always return a boolean. I get it now after closer inspection but maybe return `{isSorted: false, sortedList}` ?

Reply:
That does seem better. Updated.